Update documentation for new developers (#256)
UCLH-Foundry/PIXL: pixl_core/README.md

different PIXL services. Currently, we define two queues:

1. `pacs` for downloading and de-identifying images
2. `ehr` for downloading and de-identifying the EHR data

The image anonymisation will be triggered automatically once the image has been downloaded to the raw Orthanc server.

### RabbitMQ

RabbitMQ is used for the queue implementation.

The client of choice for RabbitMQ at this point in time is
[pika](https://pika.readthedocs.io/en/stable/), which provides both a synchronous and asynchronous
way of transferring messages. The former is geared towards high data throughput whereas the latter
is geared towards stability.  The asynchronous mode of transferring messages is a lot more complex
as it is based on the [asyncio event loop](https://docs.python.org/3/library/asyncio-eventloop.html).

### OMOP ES files

Public parquet exports from OMOP ES that should be transferred outside the hospital are copied to
the `exports` directory at the repository base.

Within this directory each project has a directory, with all extracts stored in `all_extracts` and
the `latest` directory contains a symlink to the most recent extract. This symlinking means that

> during the export stage it is clear which export should be sent.

Reviewer: All exports should specify explicitly which extract is to be exported. The "latest" symlink is just a convenience for humans AIUI.

Author: @jeremyestein happy for you to make a suggestion of how you would describe it :)

> the `latest` directory contains a symlink to the most recent extract. This symlinking means that
> during the export stage it is clear which export should be sent.

Reviewer: ```suggestion
for convenience the `latest` directory is a symlink to the most recent extract.
```
`latest` does not contain a symlink; it is one.
